lib/jsdom/living/helpers/form-controls.js

```diff
@@ -132,6 +132,9 @@ exports.isLabelable = node => {
     case "input":
       return node.type !== "hidden";
   }
+  if (node._ceDefinition?.formAssociated) {
+    return true; // as per https://html.spec.whatwg.org/#category-label
```

Comment:
Nit: this comment is redundant with the one before the function declaration, so we can omit it.